packages/perseus-editor/src/hint-editor.tsx

```diff
@@ -30,11 +30,11 @@ type HintEditorProps = {
     apiOptions?: APIOptions;
     className: string;
     imageUploader?: ImageUploader;
-    showMoveButtons?: boolean;
-    showRemoveButton?: boolean;
-    showTitle?: boolean;
-    content?: string | null | undefined;
-    replace?: boolean | null | undefined;
+    showMoveButtons: boolean;
+    showRemoveButton: boolean;
+    showTitle: boolean;
+    content: string | null;
+    replace: boolean | null;
```

Comment:
Any prop that appears in `defaultProps` should be non-optional here as TypeScript will make it optional automatically.